bzt/engine.py

```diff
@@ -320,6 +320,9 @@ def create_artifacts_dir(self, existing_artifacts=(), merged_config=None):
         for artifact in existing_artifacts:
             self.existing_artifact(artifact)
 
+    def is_functional_mode(self):
+        return self.config.get(SETTINGS).get("aggregator", None) == "functional-aggregator"
```

Comment:
Upd: `engine` can't depend on `functional` because that would create a dependency loop. So we're stuck with strings.